src/main/java/at/ac/tuwien/kr/alpha/common/atoms/ExternalLiteral.java

```diff
@@ -68,14 +71,15 @@ public ExternalLiteral substitute(Substitution substitution) {
 
 	@Override
 	public Set<VariableTerm> getBindingVariables() {
-		// If the external atom is negative, then all variables of input and output are non-binding
+		// If the external atom is negative, then all variables of input and output are
+		// non-binding
 		// and there are no binding variables (like for ordinary atoms).
 		// If the external atom is positive, then variables of output are binding.
```

Comment:
Please either reformat all, or none of those lines.

Reply:
fixed